static/app/components/featureFeedback/feedbackModal.tsx

```diff
@@ -0,0 +1,154 @@
+import {Fragment, useMemo, useState} from 'react';
+import {css} from '@emotion/react';
+import {BrowserClient} from '@sentry/react';
+
+import {addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import Button from 'sentry/components/button';
+import Textarea from 'sentry/components/forms/controls/textarea';
+import SelectField from 'sentry/components/forms/selectField';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import OrganizationStore from 'sentry/stores/organizationStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {defined} from 'sentry/utils';
+import {useLocation} from 'sentry/utils/useLocation';
+import useProjects from 'sentry/utils/useProjects';
+
+import ButtonBar from '../buttonBar';
+import Field from '../forms/field';
+
+const feedbackClient = new BrowserClient({
+  // feedback project under Sentry organization
+  dsn: 'https://3c5ef4e344a04a0694d187a1272e96de@o1.ingest.sentry.io/6356259',
+});
+
+export interface FeedBackModalProps {
+  featureName: string;
+  feedbackTypes: string[];
+}
+
+interface Props
+  extends FeedBackModalProps,
+    Pick<ModalRenderProps, 'Header' | 'Body' | 'Footer' | 'closeModal'> {}
+
+type State = {additionalInfo?: string; subject?: number};
+
+export function FeedbackModal({
+  Header,
+  Body,
+  Footer,
+  closeModal,
+  feedbackTypes,
+  featureName,
+}: Props) {
+  const {organization} = useLegacyStore(OrganizationStore);
+  const {projects, initiallyLoaded: projectsLoaded} = useProjects();
+  const location = useLocation();
+  const {user} = useLegacyStore(ConfigStore);
+
+  const [state, setState] = useState<State>({
+    subject: undefined,
+    additionalInfo: undefined,
+  });
+
+  const project = useMemo(() => {
+    if (projectsLoaded && location.query.project) {
+      return projects.find(p => p.id === location.query.project);
+    }
+    return undefined;
+  }, [projectsLoaded, location.query.project]);
+
+  function handleSubmit() {
+    const {subject, additionalInfo} = state;
+
+    if (!defined(subject)) {
+      return;
+    }
+
+    feedbackClient.captureEvent({
+      message: additionalInfo?.trim()
```

Comment:
Should we set the level for this message/exception so that it isn't an 'error'. We could use info/notice levels instead.

Reply:
added 👍